.github/workflows/dev-build.yml

```diff
@@ -224,6 +224,13 @@ jobs:
           PATHS: /*
         run: aws cloudfront create-invalidation --distribution-id "$DISTRIBUTION" --paths "$PATHS"
 
+      - name: Notify PRs about deployment
+        run: |
+          gh pr list -S "$GITHUB_SHA -is:merged" --json number --jq '.[].number' | xargs -i gh pr comment {} --body "Dev build for $GITHUB_SHA was deployed to: $DEPLOYMENT_URL" || true
```

Comment:
Oh neat, I did not know you could execute the GitHub CLI here. That opens up some opportunities 🤔

Reply:
@schalkneethling FWIW This is documented here: https://docs.github.com/en/actions/using-workflows/using-github-cli-in-workflows****